node-module-system/add.js

```diff
@@ -0,0 +1,4 @@
+export default function add(a, b) {
+  const sum = a + b;
+  console.log('Result: ', sum);
```

Comment:
Instead of logging in this function, it would be best to make a more reusable function (that is, one that can be used in multiple places) by removing the output from this function. That is, instead of logging, just return the sum and let the caller log it. (A different caller, for example, might decide to display it graphically.)